esphome/components/htu31d/htu31d.cpp

```diff
@@ -0,0 +1,254 @@
+/*
+ * This file contains source code derived from Adafruit_HTU31D which is under
+ * the BSD license:
+ *   Written by Limor Fried/Ladyada for Adafruit Industries.
+ *   BSD license, all text above must be included in any redistribution.
+ *
+ * Modifications made by Mark Spicer.
+ */
+
+#include "htu31d.h"
+#include "esphome/core/log.h"
+#include "esphome/core/hal.h"
```

Comment:
Required for `encode_uint16`
```suggestion
#include "esphome/core/hal.h"
#include "esphome/core/helpers.h"
```